Nsteps_timeout: 20
time_sleep_ms: 500
verbosity: true
P_gains: [40.0,40.0,40.0,40.0,40.0,40.0,40.0,40.0,40.0,40.0,40.0,40.0]
D_gains: [2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0]